src/sonic-yang-models/yang-models/sonic-vlan-sub-interface.yang

```diff
@@ -36,9 +36,11 @@ module sonic-vlan-sub-interface {
                 key "name";
 
                 leaf name {
-                    must "substring-before(current(), '.') = /port:sonic-port/port:PORT/port:PORT_LIST/port:name"
+                    must "(substring-before(current(), '.') = /port:sonic-port/port:PORT/port:PORT_LIST/port:name) or " +
+                         "(starts-with(substring-before(current(), '.'), 'Eth') and " +
```

Comment:
should we not check for an exact match for 'Eth' instead of starts-with?

Reply:
This is implicitly checked in LINE41.
If the sub intf name is `Eth40.100`, LINE41 will ensure the port `Ethernet40` with port index as `40` substring between `Eth` and `.` is existed in the port list. If the sub intf name is something like `Ether40.100`, it will fail the check.